nautobot/core/graphql/__init__.py

```diff
@@ -16,3 +24,24 @@ def convert_field_to_string(field, registry=None):
 def convert_field_to_string(field, registry=None):  # noqa: F811
     """Convert BinaryField to String."""
     return graphene.String()
+
+
+def execute_query(query, variables=None, request=None, user=None):
```

Comment:
Please add docstrings to both of these functions. IIRC you explained this function to me as being added to facilitate plugin usage? Perhaps it needs to be documented in one of the `docs/plugins/` files so that plugin authors can be made aware that it exists?